### ISSUE TITLE
RabbitMQ <> Python communication issue

### ISSUE THREAD

**Issue by ktamiola** [nestjs/nest#2337]
## Bug Report
I am trying to establish  `nestjs` <> `python` communication. The `netjs` part will handle the API calls, whereas the `python` worker will respond to RMQ calls. I can call the Python worker with absolutely no problem, however, when the data gets sent back to `netjs`, reply consumer is already set.

Is there an extensive 

## Current behavior
```
(node:4000) UnhandledPromiseRejectionWarning: Error: Operation failed: BasicConsume; 406 (PRECONDITION-FAILED) with message "PRECONDITION_FAILED - reply consumer already set"
    at reply (/Users/kamil/Development/peptone/cassandra-api/app/server/node_modules/amqplib/lib/channel.js:128:29)
    at ConfirmChannel.C.accept (/Users/kamil/Development/peptone/cassandra-api/app/server/node_modules/amqplib/lib/channel.js:411:7)
    at Connection.mainAccept [as accept] (/Users/kamil/Development/peptone/cassandra-api/app/server/node_modules/amqplib/lib/connection.js:64:33)
    at Socket.go (/Users/kamil/Development/peptone/cassandra-api/app/server/node_modules/amqplib/lib/connection.js:478:48)
    at Socket.emit (events.js:198:13)
    at Socket.EventEmitter.emit (domain.js:448:20)
    at emitReadable_ (_stream_readable.js:539:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
(node:4000) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:4000) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

## Input Code

```ts
import { Controller, Get, Query, Res } from '@nestjs/common';
import { Client, Transport, ClientProxy, ClientsModule, MessagePattern, EventPattern } from '@nestjs/microservices';
import { Observable } from 'rxjs';

@Controller('/v1/predict')
export class PredictController {

  @Client({
    transport: Transport.RMQ,
    options: {
      urls: [`amqp://localhost:5672`],
      queue: 'tango',
      queueOptions: { durable: false },
    },
  })
  client: ClientProxy;

  @Get()
  public getSequence(): Observable<any> {
    const pattern = { cmd: 'getSequence' };
    const data = { 'sequence': 'ADEK' };
    return this.client.send<any>(pattern, data);
  }

  // @Get()
  // async tango() {
  //   const pattern = 'tango' ;
  //   const payload = {'sequence': 'ADEK'};
  //   const result = await this.client.emit('tango', 'KAKAKAK').toPromise();
  //   // const result = await this.client.send(pattern, payload).toPromise();
  //   return result;
  // }

}
```
**Python worker code:**

```python
# The response function
def on_request(ch, method, props, body):

    # Decode the body
    request = json.loads(body)

    if request:
        logger.info('Got request {}'.format(request))

    logger.info(request['data'])

    # We may want to unpack the body JSON here
    response = worker(sequence=request['data']['sequence'])

    logger.info(response)

    ch.basic_publish(exchange='',
                     routing_key=props.reply_to,
                     properties=pika.BasicProperties(correlation_id = \
                                                props.correlation_id),
                     body=str({'pattern': 'tango', 'data' : response }))

    ch.basic_ack(delivery_tag = method.delivery_tag)

    logger.info("Done\n")

#
# Connect to RabbitMQ
#

credentials = pika.PlainCredentials('guest', 'guest')
connection = pika.BlockingConnection(pika.ConnectionParameters(host=RABBITMQ_HOST, port=5672))

if connection:
    logger.info('Connected to {}:{}'.format(RABBITMQ_HOST, 5672))

channel = connection.channel()
channel.queue_declare(queue=QUEUE)

#
# Prepare to generate the response
#

channel.basic_qos(prefetch_count=1)
channel.basic_consume(on_request, queue=QUEUE)

if channel:
    logger.info('Listening to queue="{}"'.format(QUEUE))

#
# Start listening
#

channel.start_consuming()
```

## Expected behavior
I was expecting to be able to recover the contents of the object that is sent back from Python worker to `nestjs` instance.

## Environment

<pre><code>
Nest version: 
├─┬ @nestjs/common@6.2.0
├─┬ @nestjs/core@6.2.0
├─┬ @nestjs/microservices@6.2.4
├── @nestjs/mongoose@6.1.2
├─┬ @nestjs/platform-express@6.2.0
├─┬ @nestjs/testing@6.2.0

For Tooling issues:
- Node version: v10.16.0
- Platform:  OSX 10.13.6
</code></pre>

**Comment by kamilmysliwiec** [comment]
Could you please provide a minimal repository which reproduces your issue? (that includes above code snippets)

**Comment by kamilmysliwiec** [comment]
Fixed in 6.10.9
